### PR TITLE
UBERF-7934 Update activity from collaborator

### PR DESCRIPTION
**Pull request by aonnikov**
Move activity update for collaborative docs from activity triggers to collaborator

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NmM3MWMzOWFkNTkzZjUzZjcxNzI1NzEiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.d11S8i6ameqCTWYIn5SU1F_eh3vN7a5nKu_QdR-nBm4">Huly&reg;: <b>UBERF-7925</b></a></sub>

### CODE REVIEW

File: server/collaborator/src/platform.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ import { createClient, getTransactorEndpoint } from '@hcengineering/server-clien
 import { Token, generateToken } from '@hcengineering/server-token'
 
 async function connect (token: string): Promise<Client> {
-  const endpoint = await getTransactorEndpoint(token)
+  const endpoint = await getTransactorEndpoint(token, 'external')
```
Comment (haiodo) [inline server/collaborator/src/platform.ts]: please use internal

File: server/collaborator/src/storage/adapter.ts [hunk 2]
```diff
@@ -20,5 +20,14 @@ import { Context } from '../context'
...
   loadDocument: (ctx: MeasureContext, documentId: DocumentId, context: Context) => Promise<YDoc | undefined>
-  saveDocument: (ctx: MeasureContext, documentId: DocumentId, document: YDoc, context: Context) => Promise<void>
+  saveDocument: (
+    ctx: MeasureContext,
+    documentId: DocumentId,
+    document: YDoc,
+    context: Context,
+    markup: {
+      old: Record<string, string>
+      new: Record<string, string>
```
Comment (haiodo) [inline server/collaborator/src/storage/adapter.ts]: please do not use new as name

**Review by haiodo — CHANGES_REQUESTED** [review]

File: server/collaborator/src/platform.ts (hunk 1 above)
Comment (aonnikov) [reply server/collaborator/src/platform.ts]: ahh, debugging again :(

File: server/collaborator/src/storage/adapter.ts (hunk 2 above)
Comment (aonnikov) [reply server/collaborator/src/storage/adapter.ts]: replaced with prev/next

**Review by kristina-fefelova — APPROVED** [review]

**Review by haiodo — APPROVED** [review]

**Pull request merged**